src/libraries/System.Net.Http/src/System/Net/Http/HttpClientHandler.cs

```diff
@@ -22,7 +22,7 @@ public partial class HttpClientHandler : HttpMessageHandler
     {
         private readonly HttpHandlerType _underlyingHandler;
 
-        private HttpMessageHandler Handler
+        private HttpHandlerType Handler
 #if TARGET_BROWSER
```

Comment:
To fix the error, you might change this to be more like:
```C#
#if TARGET_BROWSER
    private HttpMessageHandler Handler { get; }
#else
    private HttpHandlerType Handler => _underlyingHandler;
#endif
```
(although this approach with TARGET_BROWSER in general should be revisited, but that pre-exists your change)